WordSearch.java

```diff
@@ -0,0 +1,113 @@
+import java.util.*;
+
+/**
+ * Created by Alice Colt.
+ */
+public class WordSearch {
+
+
+    // Search for a WORD - using list
```

Comment:
This comment really doesn't add anything. Consider what comments would be useful to someone who's reading the code without any context. Generally concentrate on what it tries to do, and why it does it that way, rather than how. Don't add a comment that says the same thing as the next line of code.